src/libraries/MultisigTaskPrinter.sol

```diff
@@ -92,19 +92,24 @@ library MultisigTaskPrinter {
 
     /// @notice Prints encoded transaction data with formatted header and footer and instructions for signers.
     /// @param dataToSign The encoded transaction data to sign.
-    function printEncodedTransactionData(bytes memory dataToSign) internal pure {
+    function printEncodedTransactionData(bytes memory dataToSign) internal view {
         // NOTE: Do not change the vvvvvvvv and ^^^^^^^^ lines, as the eip712sign tool explicitly
         // looks for those specific lines to identify the data to sign.
         printTitle("DATA TO SIGN");
-        console.log("vvvvvvvv");
-        console.logBytes(dataToSign);
-        console.log("^^^^^^^^\n");
+        // 'PRINT_DATA_TO_SIGN' is set when the task is the last task in the stack.
+        if (Utils.isFeatureEnabled("PRINT_DATA_TO_SIGN")) {
```

Comment:
Can I change this to `SUPPRESS_DATA_TO_SIGN_PRINT`, and have it always print unless it's set to `true`. The only situation where it'll not be true is in stacked signing when it's not the last task.